src/pages/DirectMessage/components/MessageGroupList/MessageProfile.tsx

```diff
@@ -0,0 +1,34 @@
+import standard from "@/assets/standard.jpeg"
+import * as S from "./MessageList.Styles"
+import seenindicator from "@/assets/seenindicator.png"
+
+interface MessageProfileProps {
+  isOnline?: boolean
+  profileImg: string
+}
+const MessageProfile = ({ isOnline, profileImg }: MessageProfileProps) => {
+  return (
+    <S.MessageProfileContainer>
+      {profileImg ? (
+        <S.MessageProfileImg
+          src={profileImg}
+          alt="DMList Profile Image"
+        />
+      ) : (
+        <S.MessageProfileImg
+          src={standard}
+          alt="DMList Profile Standard Image"
+        />
+      )}
+
+      {isOnline && (
+        <S.SeenIndicator
+          src={seenindicator}
+          alt="온라인 표시"
+        />
+      )}
+    </S.MessageProfileContainer>
+  )
+}
```

Comment:
[P2] 이렇게 가독성도 높일 수 있을것 같아요

```suggestion
const MessageProfile = ({ isOnline, profileImg }: MessageProfileProps) => {
 1. const checkedImg = profileImg ? profileImg : standard
 2. const chechedImg = profileImg || standard

  return (
    <S.MessageProfileContainer>
        <S.MessageProfileImg
          src={chechedImg}
          alt="DMList Profile Image"
        />

      {isOnline && (
        <S.SeenIndicator
          src={seenindicator}
          alt="온라인 표시"
        />
      )}
    </S.MessageProfileContainer>
  )
}
```